server/controllers/recipes.js

```diff
@@ -20,48 +32,50 @@ export default {
           { userId: id }
         ]
       },
-    }))
-      .then((RecipeFound) => {
-        if (RecipeFound) {
+    })
+      .then((recipeFound) => {
+        if (recipeFound) {
           return res.status(409).json({
             error: 'You cannot create same recipe twice'
           });
         }
         return Recipe
           .create({
-            recipeName: req.body.recipeName,
-            description: req.body.description,
-            imageUrl: req.body.imageUrl,
-            ingredients: req.body.ingredients,
-            preptime: req.body.preptime,
-            servings: req.body.servings,
-            userId: userDetail.id
+            recipeName,
+            description,
+            imageUrl,
+            ingredients,
+            preptime,
+            servings,
+            userId: id
           })
-          .then(newRecipe => res.status(201).json({
-            recipeName: newRecipe.recipeName,
-            description: newRecipe.description,
-            ingredients: newRecipe.ingredients,
-            imageUrl: newRecipe.imageUrl,
-            preptime: newRecipe.preptime,
-            servings: newRecipe.servings,
-            id: newRecipe.id,
-            createdAt: moment().calendar(),
-            userId: newRecipe.userId
-          }));
+          .then(newRecipe =>
+            res.status(201).json({
```

Comment:
Seeing that the properties on `newRecipe` are the same name as that sent in the payload, I believe this can be improved to remove the repeating `newRecipe.xxx` (maybe and Object.assign or something, can't think right now 😄)

Reply:
It's also repeated above in `create`, it can be DRYied.